netci.groovy

```diff
@@ -2333,14 +2481,22 @@ Constants.allScenarios.each { scenario ->
                     }
                     else {
                         // Non-Windows
+                        def validLinuxArm64Scenarios = [ 
```

Comment:
I think you should hoist this up next to `validArmWindowsScenarios`, as part of `Constants`